resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DepsTestsSuite.scala

```diff
@@ -259,20 +294,46 @@ private[spark] trait DepsTestsSuite { k8sSuite: KubernetesSuite =>
     }
   }
 
-  private def createS3Bucket(accessKey: String, secretKey: String, endPoint: String): Unit = {
+  private def getS3Client(
+      s3Endpoint: String,
+      accessKey: String = ACCESS_KEY,
+      secretKey: String = SECRET_KEY): AmazonS3Client = {
+    val credentials = new BasicAWSCredentials(accessKey, secretKey)
+    val s3client = new AmazonS3Client(credentials)
+    s3client.setEndpoint(s3Endpoint)
+    s3client
+  }
+
+  private def createS3Bucket(
+      s3Endpoint: String = getServiceUrl(svcName),
+      bucket: String = BUCKET): Unit = {
```

Comment:
Does this PR need to change this function signature when we use always `BUCKET` during the tests? If not, please move this out from this bug-fix PR. You can do this kind of enhancement separately in a different PR.